pkg/network/dns_parser.go

```diff
@@ -183,13 +184,13 @@ func (*dnsParser) extractCNAME(domainQueried []byte, records []layers.DNSResourc
 
 func (*dnsParser) extractIPsInto(alias, domainQueried []byte, records []layers.DNSResourceRecord, t *translation) {
 	for _, record := range records {
-		if record.Type != layers.DNSTypeA || record.Class != layers.DNSClassIN {
+		if (record.Type != layers.DNSTypeA && record.Type != layers.DNSTypeAAAA) || record.Class != layers.DNSClassIN {
 			continue
 		}
 
 		if bytes.Equal(domainQueried, record.Name) ||
 			(alias != nil && bytes.Equal(alias, record.Name)) {
-			t.add(util.AddressFromNetIP(record.IP))
+			t.add(util.AddressFromNetIP(record.IP), time.Duration(record.TTL)*time.Second)
```

Comment:
Do we have an idea what the usual TTL might be for a DNS record? & if it's significantly longer than the current 3 min TTL, do we have an estimate for the impact this change might have on cache size?

Reply:
30s - 1hour is the usual range I've seen TTLs be in.